src/generated/PiecewiseFunction.jl

```diff
@@ -2,31 +2,35 @@
 This file is auto-generated. Do not edit.
 =#
 """
-    mutable struct PiecewiseFunction <: Forecast
+    mutable struct PiecewiseFunction <: TimeSeriesData
```

Comment:
When did we implement this struct? It seems redundant to what we are planning on doing for the Variable Cost.

Reply:
https://github.com/NREL-SIIP/InfrastructureSystems.jl/pull/90

Reply:
Removed.